pool/routes/routes.ml

```diff
@@ -789,6 +789,7 @@ module Root = struct
     ; post "/request-reset-password" Login.request_reset_password_post
     ; get "/reset-password" Login.reset_password_get
     ; post "/reset-password" Login.reset_password_post
+    ; get "/api/v1/status" Status.show
```

Comment:
IMO the status does not need a version namespace.

Reply:
as discussed in channel